### ISSUE TITLE
Automatically publish files based on property 

### ISSUE THREAD

**Issue by shbarton** [flowershow/flowershow#570]
As a user looking to minimise the amount of effort and friction involved with writing in Obsidian and having something published on my flowershow site, I would like to be able to publish a file by defining a certain file property. 

**Example** 
Files could be published be publish when: 
- 'Publish' checkbox is toggled on
- 'Status' tag = 'completed', or something similar (user will need to define this) 

This wouldn't need to apply to files, but specifically to files that are a part of a collection (blog, podcast, essay, etc). One way to implement this could be to let the user to choose a folder that this rule applies to. Eg. For all files that are in the /blog folder, publish files that have the the 'publish' property = true.

**Comment by rufuspollock** [comment]
@shbarton good point. So the point here would be to "auto-publish" rather than having to explicitly go into the plugin the publish the current file.

**Comment by shbarton** [comment]
@rufuspollock yeap, exactly! Or, if the publish property = true, publish that file the next time the user updates their site via the plugin.

**Comment by rufuspollock** [comment]
WONTFIX / OBSOLETE. This is old and for old flowershow self-hosted. We could open a new issue if significant interest for this in Cloud 🙂
